doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -37,6 +37,10 @@ Content:
     * [DELETE `/v1/admin/log/loggers`](#section6.5)
         * [DELETE logger by name](#section6.5.1)
         * [DELETE all loggers](#section6.5.2)
+* [Metrics](#section7)
+    * [GET `/v1/admin/metrics`](#section7.1)
+    * [DELTE `/v1/admin/metrics`](#section7.2)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1383/commits/1ca87236b3c2b0b44789f1a3d2cf6801be79fb8d